opm/simulators/aquifers/AquiferCarterTracy.hpp

```diff
@@ -92,17 +98,27 @@ class AquiferCarterTracy : public AquiferInterface<TypeTag>
     Scalar mu_w_{1}; // water viscosity
     Scalar fluxValue_{0}; // value of flux
 
+    Scalar dimensionless_time_{0};
+    Scalar dimensionless_pressure_{0};
+
     void assignRestartData(const data::AquiferData& /* xaq */) override
     {
         throw std::runtime_error {"Restart-based initialization not currently supported "
                                   "for Carter-Tracey analytic aquifers"};
     }
 
-    inline void getInfluenceTableValues(Scalar& pitd, Scalar& pitd_prime, const Scalar& td)
+    inline void getInfluenceTableValues(const Scalar td_plus_dt,
```

Comment:
Structured bindings gives a very elegant way of unpacking multiple values - I would suggest returning a `std::pair()` instead of passing mutable references.

Reply:
> would suggest returning a `std::pair()` instead of passing mutable references.

I've pushed an update to do this.